index.d.ts

```diff
@@ -18,7 +18,7 @@ declare namespace unless {
          * and methods key-pairs. If the request path or path and method match, the middleware will not run. Check
          * Examples for usage.
          */
-        path?: string | string[];
+        path?: string | RegExp | (string | RegExp | { url: (string | RegExp); methods: string[]; })[];
```

Comment:
Pretty obscure, but I did see https://github.com/jfromaniello/express-unless/blob/master/index.js#L51-L61 and https://github.com/jfromaniello/express-unless/blob/master/index.js#L22-L24. Seems kind of a redundant API to support methods in two places.